packages/phrases/src/locales/en.ts

```diff
@@ -33,6 +33,7 @@ const errors = {
   user: {
     username_exists: 'The username already exists.',
     invalid_email: 'Invalid email address.',
+    email_not_exists: 'The email address is not registered yet.',
```

Comment:
nit: 'The email address is not registered.' or 'The email address has not been registered yet.'?